mods/mtg_craftguide/locale/mtg_craftguide.fr.tr

```diff
@@ -30,24 +31,24 @@ Any magenta dye=Quelconque colorant magenta
 Any dark grey dye=Quelconque colorant gris foncé
 Any dark green dye=Quelconque colorant vert foncé
 # Label for group ingredients
-G=
+G=G
 Any item belonging to the group(s): @1=Tout item appartenant au(x) groupe(s) : @1
-Unknown Item=
-Fuel=
+Unknown Item=Object inconnu
+Fuel=Combustible
 Usage @1 of @2=Usage @1 sur @2
 Recipe @1 of @2=Recette @1 sur @2
-Previous recipe=
-Next recipe=
-Recipe is too big to be displayed.=
+Previous recipe=Recette précédente
+Next recipe=Prochaine recette
+Recipe is too big to be displayed.=La recette est trop grande pour être affichée.
```

Comment:
Shorten the translation a bit to make sure it fits on small displays or with large font sizes:

```suggestion
Recipe is too big to be displayed.=Recette trop grande pour être affichée.
```